### DISCUSSION TITLE
Não estou conseguindo achar o erro na publicação MoneyExtension

### DISCUSSION THREAD

**Discussion by sebastiaosiqueira** [balta-io/forum#711] [Tira Dúvidas] [answered]
Fundamentos do Azure, Git, GitHub e DevOps
 Publicando o pacote com GitHub Actions

Erro:
Github
Publish 
Run dotnet nuget push "MoneyExtension/bin/Release/*.nupkg" -k *** -s https://api.nuget.org/v3/index.json
error: File does not exist (MoneyExtension/bin/Release/*.nupkg).
Error: Process completed with exit code [1](https://github.com/sebastiaosiqueira/MoneyExtyensions/actions/runs/12186862436/job/33996305914#step:7:1).

![sln](https://github.com/user-attachments/assets/c76541e0-9d7e-4c6e-af0b-7c23fd427a7b)

**Comment by andrebaltieri** [comment, marked as answer]
Tem mais detalhes do erro, em qual linha dá este erro? Talvez o log do action depois de executado por favor!

**Comment by sebastiaosiqueira** [comment]
- name: Publish
 aqui esta o erro: ->>>         run: dotnet nuget push "MoneyExtensions/bin/Release/*.nupkg" -k ${{secrets.NUGET_TOKEN}} -s https://api.nuget.org/v3/index.json

**Reply by andrebaltieri** [reply]
@sebastiaosiqueira provavelmente deve ter mudado nas novas versões...
Consegue adicionar um passo a mais no YAML para listar os arquivo (`ls`) e assim vermos como está?

Acredito que deve ser:
`dotnet nuget push "MoneyExtension/bin/Release/net8.0/.nupkg"`

Mas é legal checar certinho!
